### PR TITLE
Update API documentation URLs

### PR DESCRIPTION
**Pull request by ehmicky**
**- Summary**

The API documentation URLs are about to change due to https://github.com/netlify/open-api/pull/182.

Note: this PR should be merged at the same time as https://github.com/netlify/open-api/pull/182, otherwise links will be broken.

**- Description for the changelog**

Update API documentation URLs.

### CODE REVIEW

File: src/commands/addons/config.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class AddonsConfigCommand extends Command {
       return false
     }
 
-    // TODO update getAddonManifest to https://open-api.netlify.com/#/default/showServiceManifest
+    // TODO update getAddonManifest to https://open-api.netlify.com/#/operation/showServiceManifest
```
Comment (verythorough) [inline src/commands/addons/config.js]: It looks like these links don't work with the `/` after the `#`
```suggestion
    // TODO update getAddonManifest to https://open-api.netlify.com/#operation/showServiceManifest
```

File: src/commands/addons/config.js [hunk 2]
```diff
@@ -186,7 +186,7 @@ async function update({ addonName, currentConfig, newConfig, settings, accessTok
 
   let updateAddonResponse
   try {
-    // TODO update updateAddon to https://open-api.netlify.com/#/default/updateServiceInstance
+    // TODO update updateAddon to https://open-api.netlify.com/#/operation/updateServiceInstance
```
Comment (verythorough) [inline src/commands/addons/config.js]:
```suggestion
    // TODO update updateAddon to https://open-api.netlify.com/#operation/updateServiceInstance
```

File: src/commands/addons/create.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ class AddonsCreateCommand extends Command {
 async function createSiteAddon({ addonName, settings, accessToken, siteData, error }, logger) {
   let addonResponse
   try {
-    // TODO update to https://open-api.netlify.com/#/default/createServiceInstance
+    // TODO update to https://open-api.netlify.com/#/operation/createServiceInstance
```
Comment (verythorough) [inline src/commands/addons/create.js]:
```suggestion
    // TODO update to https://open-api.netlify.com/#operation/createServiceInstance
```

File: src/commands/addons/delete.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ class AddonsDeleteCommand extends Command {
     }
     let addonResponse
     try {
-      // TODO update deleteAddon to https://open-api.netlify.com/#/default/deleteServiceInstance
+      // TODO update deleteAddon to https://open-api.netlify.com/#/operation/deleteServiceInstance
```
Comment (verythorough) [inline src/commands/addons/delete.js]:
```suggestion
      // TODO update deleteAddon to https://open-api.netlify.com/#operation/deleteServiceInstance
```

File: src/commands/addons/list.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ class AddonsListCommand extends Command {
 
     const siteData = await api.getSite({ siteId })
 
-    // TODO update getAddons to https://open-api.netlify.com/#/default/getServices
+    // TODO update getAddons to https://open-api.netlify.com/#/operation/getServices
```
Comment (verythorough) [inline src/commands/addons/list.js]:
```suggestion
    // TODO update getAddons to https://open-api.netlify.com/#operation/getServices
```

**Comment by ehmicky** [comment]
Thanks @verythorough, good catch! I've fixed them.

**Pull request merged**